oracle/README.md

```diff
@@ -34,13 +37,36 @@ oracle/
 
 TODO: need UI for this to be relevant
 
+### Node
+In order to run the Oracle node make sure you are in the root of this project i.e `/path/to/oracle/<you are here>`
+
+Copy the `.env.example` file into a new file called `.env`.  Then insert your api key into the `API_URL`
+
+**_Note:_** You do not need an api key to run any tests
+
+
+Then start a local fuel-core instance
+```bash
+fuel-core run --chain packages/node/.chainConfig.json
+```
+
+Then deploy an instance of the Oracle contract
+```bash
+forc deploy --path packages/contract --url localhost:4000 --unsigned
+```
+
+Finally start the Oracle node
+```bash
+cargo run
+```
+
 ### Tests
 In order to run the tests make sure you are in the root of this project i.e. `/path/to/oracle/<you are here>`
 
 Run the tests with the following command.
 
 ```bash
-forc test
+cargo test
```

Comment:
Since a workspace is being used I hope this ensures that all the tests can be run. I'm getting a bit confused with the CI using `forc test`, the later releases deprecating that command and using `cargo test` (while we still have not bumped the repo to reflect this) and now this uses `cargo test` too.

Reply:
Cargo test ensures all tests are run.  However, on a second look I don't even think the CI is running the oracle node tests because Cargo test is never called.